.github/workflows/update-flake.yml

```diff
@@ -0,0 +1,26 @@
+name: update-flake
+on:
+  workflow_dispatch: # allows manual triggering
+  schedule:
+    - cron: '45 7 * * 2' # runs weekly on Tuesday at 07:45
```

Comment:
Question: I assume this should only run on series/0.23, and we merge changes into `main`? Should we restrict the branch this workflow runs on then?

Reply:
Yeah, merging it forward makes the most sense.  If the flake ever had different sources, that would be awkward, but that's also unlikely to happen.

Will it not run by default on the default branch?  I'm not sure what the action is here.

Reply:
I kinda suspect the workflow may run at the scheduled time on every branch it exists, unless told otherwise. But gotta check the docs

Reply:
Weekly might be aggressive when we think about a couple dozen repos with a flake.  It hasn't run here in five months and I haven't been annoyed.  I care very much that it's here and only a little that it's fresh.

Reply:
> Scheduled workflows run on the latest commit on the default or base branch.

Nope, I was wrong. 👍 I don't mind it running weekly, it's just a drop in the bucket /shrug